src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx

```diff
@@ -14,15 +14,26 @@ import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
 import DetailDisplay from '../../../components/DetailDisplay';
 import { useTheme } from '@mui/material';
+import Chip from '@mui/material/Chip';
 
 interface ProjectDetailsProps {
   project: Project;
 }
 
 const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }) => {
   const theme = useTheme();
+  const teamName = project.team ? project.team.teamName : 'No Team Assigned';
```

Comment:
lets go with `No Team` instead of `No Team Assigned` just to keep it a little shorter